src/pybind/mgr/dashboard/frontend/src/app/shared/api/rgw-multisite.service.ts

```diff
@@ -108,6 +109,10 @@ export class RgwMultisiteService {
       params = params.set('replication_zone_name', replicationZoneName);
     }
 
+    if (selectedRealmName && selectedRealmName !== '') {
```

Comment:
```suggestion
    if (selectedRealmName) {
```
`''` is also  a `falsy` value in js